app/models/user.rb

```diff
@@ -3,6 +3,10 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: 'user_id'
   has_many :likes, foreign_key: 'user_id'
 
+  validates :name, presence: true
```

Comment:
- [ ] [OPTIONAL] Here if we put the validation length of the `name` 3 or more than 3, then it will be better validation.